ruby-challenges.rb

```diff
@@ -27,9 +27,13 @@ def odd_or_even (number)
 p odd_or_even(42)
```

Comment:
I can understand its hard to let go of modulo but even? might be worth exploring. 😁